torch/testing/_internal/common_methods_invocations.py

```diff
@@ -253,13 +253,20 @@ def get_operator_variant(self):
         """
         return self.operator_variant
 
-    def sample_inputs(self, device, dtype, requires_grad=False):
+    def sample_inputs(self, device, dtype, requires_grad=False, **kwargs):
         """Returns an iterable of SampleInputs.
 
         These samples should be sufficient to test the function works correctly
         with autograd, TorchScript, etc.
         """
-        return self.sample_inputs_func(self, device, dtype, requires_grad)
+
+        # TODO: Remove the try/except once all operators have sample_inputs_func with
+        #       **kwargs in their signature.
+        try:
+            samples = self.sample_inputs_func(self, device, dtype, requires_grad, **kwargs)
+        except TypeError:
+            samples = self.sample_inputs_func(self, device, dtype, requires_grad)
+        return samples
```

Comment:
Have added the try/except here. (we don't need to add try/except in test_ops)

Reply:
@mruberry can you please see if this is ok? Will rebase accordingly then. Thanks!

Reply:
Yep, this seems fine